modules/memfault-firmware-sdk/config/memfault_platform_config_extra.h

```diff
@@ -16,4 +16,4 @@
 
 #ifdef CONFIG_MEMFAULT_NCS_ETB_CAPTURE
 #define MEMFAULT_PLATFORM_FAULT_HANDLER_CUSTOM 1
-#endif
+#endif
```

Comment:
Super duper tiny nit: Looks like the end of file newline got removed so now the file looks like it was edited and renamed

Reply:
Oop my bad, fixed!e